.travis.yml

```diff
@@ -1,26 +1,47 @@
+sudo: false
+
 language: python
 
 python:
   - 2.7
   - 3.3
   - 3.4
+  - 3.5
 
 env:
-  - DJANGO="Django>=1.5,<1.6"
-  - DJANGO="Django>=1.6,<1.7"
-  - DJANGO="Django>=1.7,<1.8"
-  - DJANGO="Django>=1.8,<1.9"
+  - DJANGO=1.6
+  - DJANGO=1.7
+  - DJANGO=1.8
+
+matrix:
+  exclude:
+    - python: 3.5
+      env: DJANGO=1.6
+    - python: 3.5
+      env: DJANGO=1.7
+  include:
+    - python: 3.4
+      env: DJANGO=1.8  COVERAGE=true  COVERALLS_REPO_TOKEN=LdECqqwg7eelQx9w8gvooUZCFIaCqGZCv
+  allow_failures:
+    - env: COVERAGE=true
 
 install:
   - pip install flake8
-  - pip install -q $DJANGO
+  - pip install -q "Django>=$DJANGO,<$DJANGO.99"
```

Comment:
A correct statement is that the shorter version it will be incorrect in some edge cases, so it's really a perfectionist issue. I'm fine either way, I don't want this to become a distracting bikeshed